website/docusaurus.config.js

```diff
@@ -45,18 +45,17 @@ module.exports = {
         },
         theme: {
           customCss: [
+            path.resolve('src/css/docusaurusTheme.css'),
+            path.resolve('src/css/algoliaDocSearchTheme.css'),
+            path.resolve('static/css/custom.css'),
+            path.resolve('static/css/jest.css'),
             path.resolve('src/components/v1/legacyCSS.css'),
```

Comment:
renamed from `scss` to `css` and moved the legacy one to the bottom as it overrides some of the others (and for some reason, webpack produces a different order in the css now)